pkg/pillar/cmd/volumemgr/latchcontenthash.go

```diff
@@ -115,7 +114,7 @@ func maybeLatchContentTreeHash(ctx *volumemgrContext, status *types.ContentTreeS
 			status.ContentID, status.DisplayName, imageSha)
 		status.ContentSha256 = imageSha
 		if status.IsContainer() {
-			newName := maybeInsertSha(status.RelativeURL, imageSha)
+			newName := utils.InsertSha(status.RelativeURL, imageSha)
```

Comment:
Does utils.InsertSha insert a sha even if it is already present or require that the caller only call it is there is no sha? If not, the "maybe" part of the name still makes a ton of sense.

Reply:
It is unchanged. This was just extracting it out to `utils` to make it commonly usable.

I am fine with keeping the old name. Updating.